crates/router/src/core/errors/api_error_response.rs

```diff
@@ -18,129 +18,126 @@ pub enum ErrorType {
 #[derive(Debug, Clone, router_derive::ApiError)]
 #[error(error_type_enum = ErrorType)]
 pub enum ApiErrorResponse {
+    #[error(error_type = ErrorType::ServerNotAvailable, code = "IR_00", message = "This API is under development and will be made available soon")]
+    NotImplemented,
     #[error(
         error_type = ErrorType::InvalidRequestError, code = "IR_01",
-        message = "API key not provided or invalid API key used. Provide API key in the Authorization header or create new API key, using api-key (e.g api-key: API_KEY)."
+        message = "API key not provided or invalid API key used. Provide API key in the Authorization header using api-key (e.g api-key: API_KEY) or create new API key from dashboard"
```

Comment:
typically such details aren't provided in error message but can be provided in some error reference in the docs. (`create new API key from dashboard`)

Reply:
Have addressed this